### ISSUE TITLE
list object has not attribute _iteritems

### ISSUE THREAD

**Issue by pzwang** [conda/conda#2213]
```
[pwang@0229-hyperion ~/src/datashader.lines (lines)]$ conda create -n datashader.lines --clone datashader
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata: ....
src_prefix: '/Users/pwang/anaconda/envs/datashader'
dst_prefix: '/Users/pwang/anaconda/envs/datashader.lines'
Packages: 105
Files: 1781
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/Users/pwang/anaconda/bin/conda", line 6, in <module>
    sys.exit(main())
  File "/Users/pwang/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 139, in main
    args_func(args, p)
  File "/Users/pwang/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 146, in args_func
    args.func(args, p)
  File "/Users/pwang/anaconda/lib/python2.7/site-packages/conda/cli/main_create.py", line 49, in execute
    install.install(args, parser, 'create')
  File "/Users/pwang/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 247, in install
    clone(args.clone, prefix, json=args.json, quiet=args.quiet, index=index)
  File "/Users/pwang/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 84, in clone
    quiet=quiet, index=index)
  File "/Users/pwang/anaconda/lib/python2.7/site-packages/conda/misc.py", line 243, in clone_env
    sorted_dists = r.dependency_sort(dists)
  File "/Users/pwang/anaconda/lib/python2.7/site-packages/conda/resolve.py", line 716, in dependency_sort
    for key, value in iteritems(must_have):
  File "/Users/pwang/anaconda/lib/python2.7/site-packages/conda/compat.py", line 157, in iteritems
    return iter(getattr(d, _iteritems)())
AttributeError: 'list' object has no attribute 'iteritems'
```

**Comment by pzwang** [comment]
```
$ conda info
Using Anaconda Cloud api site https://api.anaconda.org
Current conda install:

             platform : osx-64
        conda version : 4.0.1
  conda-build version : 1.19.0
       python version : 2.7.11.final.0
     requests version : 2.9.1
```

**Comment by mcg1969** [comment]
Known issue. Can you update to 4.0.2?

**Comment by kalefranz** [comment]
Should be fixed in yesterday's `4.0.2`.  Please re-open if not.
